containers/CountdownContainer.js

```diff
@@ -0,0 +1,96 @@
+import React from 'react'
+import Countdown from 'react-countdown-now'
+
+class CountdownContainer extends React.Component {
+  state = {
+    mounted: false
+  }
+  componentDidMount () {
+    this.setState({ mounted: true })
+  }
+  render () {
+    if (this.state.mounted) {
+      return <Countdown date={'03 June 2017 09:30:00'} renderer={renderCountdown} />
+    } else {
+      return null
```

Comment:
When the component is mounted, it causes the layout to jump down. Maybe we can add some placeholder data for this element, like this:

```js
return renderCountdown({ days: '…', hours: '…', minutes: '…', seconds: '…', completed: false })
```